### PR TITLE
fix(definition-table): do not use default slot in every case

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

1. There was an issue in `DefinitionTable` that object values are not shown in tags.
That was because `DefinitionTable` always used `default` slot of `Definition` component.
`Definition` component automatically render object values as tags, but by using `default` slot of it, it didn't work.
This is an example of showing an object values in tags well.
image

2. Removed origin data and replaced them with random values using fakerjs.

### CODE REVIEW

File: packages/mirinae/src/data-display/tables/definition-table/PDefinitionTable.stories.mdx [hunk 1]
```diff
@@ -64,15 +64,21 @@ It uses the [Definition component](?path=/docs/data-display-tables-definition-ta
...
                 { label: 'Name', name: 'name' },
                 { label: 'Provider', name: 'provider' },
+                { label: 'Plugin', name: 'plugin_info' },
             ],
             data: {},
             loading: true
         });
         setTimeout(() => {
             state.data = {
-                collector_id: 'collector-6746d641c98b',
```
Comment (sulmoJ) [inline packages/mirinae/src/data-display/tables/definition-table/PDefinitionTable.stories.mdx:73]: Praise: abcdefg

**Review by sulmoJ — APPROVED** [review]
LGTM!
In the past, there used to be a PR template for Mirena, but it was inconvenient because it didn't include a checklist for Storybook. Let me add a template!

File: packages/mirinae/src/data-display/tables/definition-table/PDefinitionTable.vue [hunk 2]
```diff
@@ -33,9 +33,7 @@
                             >
                                 <slot :name="`data-${idx}`"
                                       v-bind="{...scope, index: idx, items}"
-                                >
-                                    {{ scope.value }}
-                                </slot>
```
Comment (yuda110) [inline packages/mirinae/src/data-display/tables/definition-table/PDefinitionTable.vue:38]: 🫢

**Review by yuda110 — APPROVED** [review]
LGTM!

**Comment by WANZARGEN** [comment]
@sulmoJ That's a good idea!

**Pull request merged**